examples/init_with_target_level.rs

```diff
@@ -0,0 +1,12 @@
+use log::{Level, LevelFilter};
+use simple_logger::SimpleLogger;
+
+fn main() {
+    SimpleLogger::new()
+        .with_level(Level::Info)
+        .with_target_level("init_with_target_level", LevelFilter::Off)
```

Comment:
**Bikeshedding:** I don't think the naming is really intuitive. FWIW, the `env_logger` crate uses `		.filter_level(log::LevelFilter::Trace).filter_module("rumqtt", log::LevelFilter::Info)`. I'd perhaps go with `with_level` and `with_module_level`.